src/NUnitTestExtractor/NUnitTestExtractorApp.cs

```diff
@@ -16,85 +16,66 @@ public static class NUnitTestExtractorApp
     {
         private static Options _options = new Options();
 
-        public enum Level { Namespace, Class, Function, TestCase, Null }
+        public enum Level { Namespace, Class, Function, TestCase };
 
-        static void Main(string[] args)
+        public static void Main(string[] args)
```

Comment:
[Should] Main() should return an int instead of void, then put the whole body of Main() in a try/catch block and return a negative number if you catch an exception.  On success, return 0.

Reply:
Unless the upper lvl caller will interpret the error code. But as we discussed, as long as we count the issue, we want to fail the process, correct?